scripts/zones/Yuhtunga_Jungle/mobs/Carthi.lua

```diff
@@ -15,18 +15,18 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
+    if (isKiller) then -- Causes this to only run once per Carthi kill
```

Comment:
I'm assuming there's no killer because of this: https://github.com/DarkstarProject/darkstar/blob/master/scripts/zones/Sauromugue_Champaign/mobs/Old_Sabertooth.lua#L25

...which needs to be fixed.